src/Domain/LeanCode.DomainModels/Model/SId.cs

```diff
@@ -0,0 +1,162 @@
+using System.Reflection;
+using LeanCode.DomainModels.Serialization;
+
+namespace LeanCode.DomainModels.Model
+{
+    /// <summary>A stronly typed id consisting of a guid prefixed with a lower-cased name of an entity.</summary>
```

Comment:
...consisting of a guid with its dashes skipped...

Reply:
Or do we want to stop skipping those dashes?

Reply:
I'd skip them but I see no point in being explicit about in comments, you should never rely on that...